.trunk/trunk.yaml

```diff
@@ -34,6 +34,11 @@ lint:
     - shfmt@3.6.0
     - trufflehog@3.86.0
     - yamllint@1.35.1
+  ignore:
+    - linters: [ALL]
+      paths:
+        # Ignore md files from dataset
+        - rag/rag-mdfiles/extras/**
```

Comment:
why are we ignoring these files?

Reply:
ignoring these files because they are part of a dataset used to extract a knowledge graph and we don't want to modify the original source data

Reply:
can we add `<!-- markdownlint-disable -->` at the top of the file instead? this is more scalable across many recipes

Reply:
What do you think @rderbier? Would that break your markdown parsing in the app? Could you give it a try?

Reply:
I prefer to rename it .txt
Is the tool also verifying .txt file for formating ?

Reply:
working, But I still don't get the point of checking data files. It's not code. Would you add a tag in a CSV file if it was checked with error by a commit tool?

We should exclude folders named "data" for example, so we can put our data there.

Reply:
it's generally good practice to manage by file formats rather than paths